.github/workflows/qat.yml

```diff
@@ -15,160 +15,70 @@ on:
-            const checklyFailed = ${{ steps.run-checks.outputs.checkly_failed }};
-            const checklyStatus = checklyFailed ? ':x: Some Playwright tests failed. Please review the Checkly Report to verify no breaking changes were introduced' : ':white_check_mark: All Playwright tests passed. Checkly Report';
 
-            const existingComment = await github.rest.issues.listComments({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo
-            });
+          # Set environment URLs based on the branch name
+          #
+          # The `version-hash` file is created by jenkins script when a branch is pushed
+          # https://github.com/penske-media-corp/pmc-jenkins-scripts/blob/8ee0244a8485b61cfa6d3f9af00f7e6a55ddb042/bin/push-pmcqa-repo.sh#L36-L38
+          # 
+          # Examples
+          #
+          # Feature branch example:
+          # ENVIRONMENT_URL: "https://test.sourcingjournal.pmcqa.com"
+          # HASH_URL: "https://test.sourcingjournal.pmcqa.com/wp-content-vipgo-sites/test/themes/vip/pmc-sourcingjournal-2021/version-hash"
+          #
+          # main branch example:
+          # ENVIRONMENT_URL: "https://sourcingjournal.pmcqa.com"
+          # HASH_URL: "https://sourcingjournal.pmcqa.com/wp-content-vipgo/themes/vip/pmc-sourcingjournal-2021/version-hash"
+          #
+          if [[ "$BRANCH_NAME" == "main" ]]; then
+            ENVIRONMENT_URL="https://${{ inputs.QA_SITE_DOMAIN }}"
+            HASH_URL="https://${{ inputs.QA_SITE_DOMAIN }}/wp-content-vipgo/themes/vip/${{ inputs.THEME_NAME }}/version-hash"
+          else
+            ENVIRONMENT_URL="https://${BRANCH_NAME}.${{ inputs.QA_SITE_DOMAIN }}"
+            HASH_URL="https://${BRANCH_NAME}.${{ inputs.QA_SITE_DOMAIN }}/wp-content-vipgo-sites/${BRANCH_NAME}/themes/vip/${{ inputs.THEME_NAME }}/version-hash"
+          fi
 
-            // Delete any existing Checkly Report comments
-            for (const comment of existingComment.data) {
-              if (comment.body.includes('Checkly Report')) {
-                await github.rest.issues.deleteComment({
-                  owner: context.repo.owner,
-                  repo: context.repo.repo,
-                  comment_id: comment.id
-                });
-              }
-            };
+          echo "ENVIRONMENT_URL=$ENVIRONMENT_URL" >> $GITHUB_ENV
+          echo "HASH_URL=$HASH_URL" >> $GITHUB_ENV      
 
-            // Create a new Checkly Report comment
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: `<details><summary>${checklyStatus}:</summary>\n\n${report}</details>`
-            });
+      - name: Quality Assurance Testing (QAT)
+        id: qat
+        uses: penske-media-corp/pmc-github-actions/actions/tests/checkly@feature/qat
```

Comment:
Noting for all that this PR is blocked until the dependency is merged to the default branch in `pmc-github-actions`.

Reply:
Also note, once https://github.com/penske-media-corp/pmc-github-actions/pull/32 is merged, I'll need to update this line from `@feature/qat` to `@main` prior to merging this PR

Reply:
Done in [d4d0fb5](https://github.com/penske-media-corp/github-workflows-wordpress/pull/53/commits/d4d0fb5c97e96c592e874118d0cabbeaebe24e05)